### PR TITLE
mch stores

### PR DESCRIPTION
**Pull request by Injiri**
this fix enables load mch app  into KenyaEMR 17.0.3 platform

### CODE REVIEW

File: api/pom.xml [hunk 1]
```diff
@@ -47,10 +47,15 @@
...
 		</dependency>
+        <dependency>
+            <groupId>org.openmrs.module</groupId>
+            <artifactId>metadatadeploy-api</artifactId>
+            <version>${metadatadeployVersion}</version>
+        </dependency>
 		<dependency>
 			<groupId>org.openmrs.module</groupId>
 			<artifactId>hospitalcore-api</artifactId>
-			<version>${hospitalcorVersion}</version>
```
Comment (ningosi) [inline api/pom.xml:53]: We longer need the version here? how are we supplying that then?

File: api/src/main/java/org/openmrs/module/mchapp/api/ImmunizationService.java [hunk 2]
```diff
@@ -4,14 +4,8 @@
...
 import org.openmrs.api.db.DAOException;
-import org.openmrs.module.hospitalcore.model.ImmunizationStoreDrug;
-import org.openmrs.module.hospitalcore.model.ImmunizationStoreDrugTransactionDetail;
-import org.openmrs.module.hospitalcore.model.ImmunizationStoreTransactionType;
 import org.openmrs.module.hospitalcore.model.InventoryDrug;
-import org.openmrs.module.mchapp.model.ImmunizationEquipment;
-import org.openmrs.module.mchapp.model.ImmunizationStockout;
-import org.openmrs.module.mchapp.model.ImmunizationStorePatientTransaction;
-import org.openmrs.module.mchapp.model.TransactionType;
+import org.openmrs.module.mchapp.model.*;
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/mchapp/api/ImmunizationService.java:8]: Let us just import what we need but NOt wildcards

File: api/src/main/java/org/openmrs/module/mchapp/db/hibernate/HibernateImmunizationCommoditiesDAO.java [hunk 3]
```diff
@@ -6,22 +6,18 @@
 import org.apache.commons.logging.LogFactory;
 import org.hibernate.Criteria;
 import org.hibernate.SQLQuery;
-import org.hibernate.Session;
 import org.hibernate.SessionFactory;
 import org.hibernate.cfg.NotYetImplementedException;
+import org.hibernate.classic.Session;
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/mchapp/db/hibernate/HibernateImmunizationCommoditiesDAO.java:11]: What is this import for?

Comment (ningosi) [reply api/src/main/java/org/openmrs/module/mchapp/db/hibernate/HibernateImmunizationCommoditiesDAO.java:11]: Why not use this **import org.hibernate.Session** instead?

**Review by ningosi — COMMENTED** [review]
Your work looks great, please have a look at what I have commented on

File: api/src/main/java/org/openmrs/module/mchapp/db/hibernate/HibernateImmunizationCommoditiesDAO.java (hunk 3 above)
Comment (Injiri) [reply api/src/main/java/org/openmrs/module/mchapp/db/hibernate/HibernateImmunizationCommoditiesDAO.java:11]: [An extension of the Session API, including all deprecated methods from Hibernate2. This interface is provided to allow easier migration of existing applications](https://docs.jboss.org/hibernate/core/3.3/api/org/hibernate/classic/Session.html). I agree our new code should use org.hibernate.Session.

**Review by ningosi — APPROVED** [review]

**Pull request merged**